hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsInputStream.java

```diff
@@ -696,6 +713,11 @@ public boolean hasCapability(String capability) {
     return buffer;
   }
 
+  @VisibleForTesting
+  public int getReadAheadRange() {
```

Comment:
do we also want to include this in toString()?

Reply:
We are not adding other configs so doesn't make sense for just this one.